src/app/note-details/note-details.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="flex">
+  <mat-card  class="mat-elevation-z8 ">
+    <mat-card-header>Product Information</mat-card-header>
+    <mat-form-field appearance="standard">
+      <mat-label>Note Name</mat-label>
+      <input matInput id="name" name="name" type="text" required [(ngModel)]="note.name">
```

Comment:
Challenge mode: use reactive forms for the details form now! :)

Reply:
Haha, that is part of Day 9.  So I'll be doing that today.